addons/metadata.themoviedb.org/resources/language/English/strings.po

```diff
@@ -43,3 +43,7 @@ msgstr ""
 msgctxt "#30006"
 msgid "Preferred Certification Country"
 msgstr ""
+
+msgctxt "#30007"
+msgid "Search for adult movies too"
```

Comment:
I don't like the wording, should we change it to something like "Include adult movies"